pages/start.tsx

```diff
@@ -8,7 +8,7 @@ import ChooseTimezoneIcon from "img/icons/choose-timezone.svg";
 import ChooseTimezoneActiveIcon from "img/icons/choose-timezone-active.svg";
 import ChooseInterestsIcon from "img/icons/choose-interests.svg";
 import ChooseInterestsActiveIcon from "img/icons/choose-interests-active.svg";
-import HighlightIcon from "img/icons/highlight-icon.svg";
+import HighlightIcon from "img/icons/highlixfght-icon.svg";
```

Comment:
think this is a typo :D

Reply:
oh sorry, I edited from github.com directly and missed it :/
It's fixed